src/main/java/com/google/sps/travelbud/CountryServlet.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.travelbud;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// hard coded data
+class Country {
+  int id;
+  String name;
+  String CultureNorms;
+  String Languages;
+
+  Country(int id, String name, String CultureNorms, String Languages) {
+    this.id = id;
+    this.name = name;
+    this.CultureNorms = CultureNorms;
+    this.Languages = Languages;
+  }
+}
+/** Servlet that returns some example content. */
+@WebServlet(urlPatterns = {"/api/countries/*"})
+public class CountryServlet extends HttpServlet {
+  @Override
+
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
In general I have similar remarks for this file as in the `CityServlet.java` file